crates/sdk/src/default_main.rs

```diff
@@ -228,8 +228,11 @@ where
     <Setup::Connector as Connector>::Configuration: Clone,
     <Setup::Connector as Connector>::State: Clone,
 {
-    init_tracing(&serve_command.service_name, &serve_command.otlp_endpoint)
-        .expect("Unable to initialize tracing");
+    init_tracing(
```

Comment:
What's the rationale for prefering to `.as_deref()` over `&` ?

Reply:
It transforms `Option<String>` to `Option<&str>`.